eZ/Publish/Core/Repository/RoleService.php

```diff
@@ -434,6 +825,32 @@ public function loadRole($id)
     }
 
     /**
+     * Loads a role draft for the given identifier.
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException if the authenticated user is not allowed to read this role
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException if the authenticated user is not allowed to read this role
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException if a role with the given name was not found
+     *
+     * @param string $identifier
+     *
+     * @return \eZ\Publish\API\Repository\Values\User\RoleDraft
+     */
+    public function loadRoleDraftByIdentifier($identifier)
```

Comment:
@andrerom Editing conflicts are not detected yet, there's a followup for that (not decided how we should deal with them UI-wise). Currently I'm just overwriting any existing drafts.

> it only checks against published items for identifier and remote id conflicts

I don't see how that can work here - the DB will refuse to store on draft conflicts, and we need to store drafts when editing...?

So, like this then?
- remove loadRoleDraftByIdentifier() from the RoleService, instead call SPI method directly
- on conflict, just throw exceptions in RoleService, and catch them in PlatformUI